.github/workflows/workflow.yaml

```diff
@@ -32,7 +34,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest] # TODO: this job does not yet work on any version of macOS
+        os: 
+          - ubuntu-latest # TODO: this job does not yet work on any version of macOS
```

Comment:
Perhaps we can remove the comment, seems like it no longer hold strue.

Reply:
Good point, will do.